wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -100,11 +105,19 @@ def __init__(
             if resp.get("state") == SchedulerState.CANCELLED.name:
                 self._state = SchedulerState.CANCELLED
             self._sweep_config = yaml.safe_load(resp["config"])
+            self._num_runs_launched: int = len(resp["runs"])
+            if self._num_runs_launched > 0:
+                wandb.termlog(
+                    f"{LOG_PREFIX}Found {self._num_runs_launched} previous runs for sweep {self._sweep_id}"
+                )
```

Comment:
@KyleGoyette Not positive this works in practice, but it **should** be all thats required! this is a much better path, thanks for the comment.